core/domain/classifier_services.py

```diff
@@ -124,6 +124,57 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def train(exploration):
```

Comment:
I think what's needed is more of a design doc, actually. See if you can talk to @prasanna08 and figure out what the flow actually is. This PR may need some underlying infrastructure (like the jobs queue) to be built first.